services-core/src/orderbook/streamed/updating_orderbook.rs

```diff
@@ -73,6 +73,11 @@ impl UpdatingOrderbook {
                             "Failed to construct orderbook from path (using default): {:?}",
                             error,
                         );
+                    } else {
+                        warn!(
```

Comment:
I don't think this should be a warning, but an `info`. The reason I say this is that you don't expect this file to be there the first time you run, and showing a warning might be a little strong.